sdk/examples/xo_python/sawtooth_xo/processor/handler.py

```diff
@@ -62,6 +62,9 @@ def apply(self, transaction, state_store):
         if name == "":
             raise InvalidTransaction("Name is required")
 
+        if '|' in name:
+            raise InvalidTransaction('Name cannot contain "|"')
```

Comment:
Because commas are used in the payload serialization, I don't think it's possible to get a comma into the name. If the payload doesn't contain exactly two commas, it should be rejected by `name, action, space = transaction.payload.decode().split(",")`